src/mpcCoreKit.ts

```diff
@@ -478,6 +466,20 @@ export class Web3AuthMPCCoreKit implements ICoreKit {
     }
   }
 
+  public async _finalizeOauthLogin(loginResponse: TorusKey, userInfo: UserInfo, persistSessionSigs = true, importTssKey?: string): Promise<void> {
+    const postBoxKey = this._getPostBoxKey(loginResponse);
+    this.torusSp.postboxKey = new BN(postBoxKey, "hex");
+
+    this.updateState({
+      postBoxKey,
+      postboxKeyNodeIndexes: loginResponse.nodesData?.nodeIndexes,
```

Comment:
do we need to take nodeIndexes in to consideration?

Reply:
yes, i need it so that passkey link api should know which nodes contains shares of the postbox key